app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -72,6 +142,24 @@ def calculate_line_8
         number_of_line_8_exemptions * 1_000
       end
 
+      def calculate_line_40a
+        is_mfs = @intake.filing_status_mfs?
+
+        case @intake.household_rent_own
+        when "own"
+          if @intake.property_tax_paid.nil?
+            return nil
```

Comment:
in this case, the `property_tax_paid` just remains `nil`? doesn't this explode when 
`(property_tax_paid / 2.0).round` or `property_tax_paid.round` is called?